examples/sklearn-gridsearchcv-replacement.ipynb

```diff
@@ -149,14 +144,77 @@
     "\n",
     "opt = BayesSearchCV(\n",
     "    pipe,\n",
-    "    [(lin_search, 16), (dtc_search, 24), (svc_search, 32)], # (parameter space, # of evaluations)\n",
+    "    [(svc_search, 20), (linsvc_search, 16)], # (parameter space, # of evaluations)\n",
```

Comment:
Reduced # of iterations and estimators in this example. Makes it a bit more simple, as well as faster to run